### PR TITLE
fixes(small): 4 small things, packaged nicely into one PR

### PR DESCRIPTION
**Pull request by pvinis**
The type of this PR is: **Enhancement**

This PR resolves [CX-]

### Description

- Fixed the UI problem with the card number in profile
Before
Screenshot 2021-02-08 at 12 47 26
After
Screenshot 2021-02-08 at 12 48 03
- add a yarn script for opening a URL in the app.
To be used like `yarn open-url artsy:///search --android`. Taken from davids notion page, and turned into a script.
- remove a fragment that I saw that was not really needed.
- add relay env provider for android, so viewing rooms work, because of relay hooks.

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "lint-fix": "yarn lint -- --fix",
     "lint": "tslint 'src/**/*.{ts,tsx}'",
     "open-sim": "open -a Simulator",
+    "open-url": "npx uri-scheme open", "//": "Use like `yarn open-url artsy:///works-for-you --android`",
```
Comment (pepopowitz) [inline package.json:33]: Sneaky way to make a JSON comment 😉

**Review by pepopowitz — APPROVED** [review]
👍

**Comment by MounirDhahri** [comment]
🙌

File: package.json (hunk 1 above)
Comment (ds300) [reply package.json:33]: rather than this comment, why don't we just have separate commands for android and ios?

**Pull request merged**